packages/brain/src/modules/apiServers/launchpad/routes/feeRecipients.ts

```diff
@@ -0,0 +1,138 @@
+import express from "express";
```

Comment:
We agreed that the launchpad API would be a proxy of the eth keymanager API so we kind of follow the same standard they have. Check out https://ethereum.github.io/keymanager-APIs/#/Fee%20Recipient

I would define the same specs for the api `/eth/v1/validator/{pubkey}/feeRecipient`

Also I would add some comments refering to these docs

Reply:
You are right, but in this case it is not possible to implement the required functionality in that way. I need to be able to retrieve the fee recipient for several validators at the same time. If I used that endpoint I would not be able to do it, as I would need to perform a call for each validator

Reply:
Then for me, the cleanest solution would be to create the same API as the standard, and then a new endpoint for retrieving all the validators fee recipients, something like `/eth/v1/validator(s)/feeRecipient`

Reply:
But that way I would need to make a call for each validator. In this way, only 1 call is enough to retrieve information about all of the validators I need

Reply:
Would it be worth it to use that endpoint and make a call for each validator for the sake of simplicity?

Since we control client and server, ratelimiting would not be a problem

Reply:
I don't think so. It would cause unnecessary overhead for any process that need to wait for this call